x-pack/plugins/security_solution/public/management/pages/event_filters/store/selector.ts

```diff
@@ -139,12 +159,27 @@ export const isCreationSuccessful = (state: EventFiltersListPageState): boolean
   return isLoadedResourceState(state.form.submissionResourceState);
 };
 
-export const getCreationError = (state: EventFiltersListPageState): ServerApiError | undefined => {
+export const isUninitialisedForm = (state: EventFiltersListPageState): boolean => {
+  return isUninitialisedResourceState(state.form.submissionResourceState);
+};
+
+export const getActionError = (state: EventFiltersListPageState): ServerApiError | undefined => {
   const submissionResourceState = state.form.submissionResourceState;
 
   return isFailedResourceState(submissionResourceState) ? submissionResourceState.error : undefined;
 };
 
+export const getSubmissionResourceState: EventFiltersSelector<
+  StoreState['form']['submissionResourceState']
+> = (state) => {
+  return state.form.submissionResourceState;
+};
+
+export const getSubmissionResource = createSelector(
```

Comment:
This selector confuses me - its returning whatever the `getSubmissionResourceState()` returns, so why do we need it?

Reply:
Yeah you are right, this is unnecessary. I'll remove it in the next PR if it's ok for you